src/main/webapp/jsp/instructorFeedbackEdit.jsp

```diff
@@ -36,6 +40,7 @@
   </c:if>
    <br>
   <input type="hidden" id="num-questions" value="${fn:length(data.qnForms)}">
+  <input type="hidden" name="${FEEDBACK_SESSION_ENABLE_EDIT}" value="${data.isToBeLoadedInEditMode}">
```

Comment:
Make it an optional parameter, and set the data attribute only if the parameter is defined.